CHANGELOG.md

```diff
@@ -9,19 +9,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Legend - label trimming when special characters are present in the labels
 
+### Changed
+- `<ChartWrapper \>` - moved all necessary chart wrapper css in ChartWrapper component
```

Comment:
the actual [self-closing tag](https://github.com/yannickcr/eslint-plugin-react/blob/HEAD/docs/rules/self-closing-comp.md) should be `<ChartWrapper />`

Reply:
I feel like this should go under unreleased for now right?